pkg/gatewayserver/io/basicstationlns/basicstationlns.go

```diff
@@ -181,7 +189,19 @@ func (s *srv) handleTraffic(c echo.Context) error {
 		"remote_addr", c.Request().RemoteAddr,
 	))
 
-	ctx, ids, err := s.server.FillGatewayContext(ctx, ttnpb.GatewayIdentifiers{GatewayID: id})
+	// Convert the ID to EUI.
+	str := euiHexPattern.FindAllString(id, -1)
+	if len(str) != 2 || len(str[1]) != 16 {
+		return errGatewayID.WithAttributes("id", id)
+	}
+	hexValue, err := hex.DecodeString(str[1])
+	if err != nil {
+		return errGatewayID.WithAttributes("id", id)
+	}
+	eui := types.EUI64{}
```

Comment:
I thought we preferred `var eui types.EUI64` in this situation? cc: @rvolosatovs?

Reply:
Ok updated. I'm neutral on this.